### PR TITLE
Create MetaData Schema

### PR DESCRIPTION
**Pull request by demoric102**
Create MetaData Schema module. 

This first tries to submit without a schema name, then without a Node description befpre finally creating the Meta Data Schema

### CODE REVIEW

File: test/BEXIS2_MetaData_Schema/dataTypesElements.js [hunk 1]
```diff
@@ -0,0 +1,162 @@
+/**
```
Comment (geofranzi) [inline test/BEXIS2_MetaData_Schema/dataTypesElements.js]: It's a copy from https://github.com/bexis/BexisTest/blob/master/test/BEXIS2_Data_Type/dataTypesElements.js. I do not see it's used here. Please remove here if not used.

File: test/BEXIS2_MetaData_Schema/create.js [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+
+    // click Next button
+    await assert.isFulfilled(page.click('#nextBt'), 'Should Click Next button');
+
+    // click generate button
+    await assert.isFulfilled(page.waitForSelector('#ReadSource > div.wizardStep_Main > div.wizardStep_Content.stepper > table > tbody > tr:nth-child(3) > td:nth-child(1) > a'));
+    await assert.isFulfilled(page.click('#ReadSource > div.wizardStep_Main > div.wizardStep_Content.stepper > table > tbody > tr:nth-child(3) > td:nth-child(1) > a'), 'should click generate button');
+
+    // click Next button
+    await assert.isFulfilled(page.click('#nextBt'), 'Should Click Next button');
```
Comment (geofranzi) [inline test/BEXIS2_MetaData_Schema/create.js]: A successful test should finish with "success". In this case, we have to check if either click on next does not have any effect or we rely on a shown error message which is easier. Same for the following tests.

File: test/BEXIS2_MetaData_Schema/create.js [hunk 3]
```diff
@@ -0,0 +1,166 @@
+import Browser from '../../util/Browser';
```
Comment (geofranzi) [inline test/BEXIS2_MetaData_Schema/create.js:1]: Please also revise all the comments. Sometimes still copy & paste from "data types" ;-). Sometimes it's missing e.g. Line 24/25.

**Review by geofranzi — CHANGES_REQUESTED** [review]
see my comments

File: test/BEXIS2_MetaData_Schema/dataTypesElements.js (hunk 1 above)
Comment (demoric102) [reply test/BEXIS2_MetaData_Schema/dataTypesElements.js]: It has now been removed

File: test/BEXIS2_MetaData_Schema/create.js (hunk 2 above)
Comment (demoric102) [reply test/BEXIS2_MetaData_Schema/create.js]: I am now console logging the success

File: test/BEXIS2_MetaData_Schema/create.js (hunk 3 above)
Comment (demoric102) [reply test/BEXIS2_MetaData_Schema/create.js:1]: Comments have been modified

**Review by geofranzi — APPROVED** [review]

**Pull request merged**